Dockerfile.ci

```diff
@@ -0,0 +1,18 @@
+ARG commit=latest
+FROM elifesciences/hypothesis-dummy:${commit}
+
+# extract into another image and put here with multi-stage build
+USER root
+RUN git clone https://github.com/asm89/smoke.sh /opt/smoke.sh
+USER www-data
+
+# extract into another image and put here with multi-stage build
```

Comment:
now installed by `www-data` because it's the user that will run it, but perhaps it can be installed by `elife` and run in read-only by `www-data`